src/eko/couplings.py

```diff
@@ -357,46 +377,50 @@ class Couplings:
 
     def __init__(
         self,
-        couplings_ref,
-        scale_ref,
-        masses,
-        thresholds_ratios,
-        order=(1, 0),
-        method="exact",
-        nf_ref=None,
-        max_nf=None,
-        hqm_scheme="POLE",
+        couplings: CouplingsRef,
+        order: Order,
+        method: CouplingEvolutionMethod,
+        masses: HeavyQuarkMasses,
+        hqm_scheme: QuarkMassSchemes = QuarkMassSchemes.POLE,
+        thresholds_ratios: MatchingScales = MatchingScales(c=1.0, b=1.0, t=1.0),
     ):
         # Sanity checks
-        if couplings_ref[0] <= 0:
-            raise ValueError(f"alpha_s_ref has to be positive - got {couplings_ref[0]}")
-        if couplings_ref[1] <= 0:
-            raise ValueError(
-                f"alpha_em_ref has to be positive - got {couplings_ref[1]}"
-            )
-        if scale_ref <= 0:
-            raise ValueError(f"scale_ref has to be positive - got {scale_ref}")
+        def positive(name, var):
```

Comment:
```suggestion
        def assert_positive(name, var):
```
with according changes below

Reply:
Done in 6321b6e503f4cb145c8e9f3df3c7386657404f5d